ethereum/contracts/Splitter.sol

```diff
@@ -2,72 +2,27 @@ pragma solidity >= 0.5.7;
 pragma experimental ABIEncoderV2;
```

Comment:
Experimental ABIencoder should be avoided. I didn't notice it is being used in any functions.

Reply:
Try to limit solidity version range to versions you are certain will work, or even set it to precisely one version, without `^`. The compiler can be the source of problems even when it seems to compile, so use it as a way to signal everyone about the version you used to develop and test.